packages/ignite-cli/src/commands/info.js

```diff
@@ -0,0 +1,45 @@
+// @cliDescription Displays info about a given Ignite plugin.
+// ----------------------------------------------------------------------------
+
+const exitCodes = require('./exitCodes')
+const validateName = require('../lib/validateName')
+const fetchPluginRegistry = require('./fetchPluginRegistry')
+
+/**
+ * Shows info about a particular plugin.
+ *
+ * @param {Object} context The gluegun context.
+ */
+const showPluginInfo = async function (context) {
+  console.log('I was called!')
```

Comment:
Hey, I just logged you.
And this is crazy!
So here's my console.
So cull me maybe?